CHANGELOG.md

```diff
@@ -1,60 +1,60 @@
-## Changelog
+<!-- markdownlint-disable-file MD024 -->
+# Changelog
+
+This project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## v0.21.0 - 2021-9-20
 
 ### Added
 
 * Transmission stats
-  - Added `:rssi_4bars` and `:rssi_dbm`
+  * Added `:rssi_4bars` and `:rssi_dbm`
 
-### Changes
+### Changed
 
 * Transmission stats
-  - `:rssi` is now `:rssi_hops` and has been changed from a tuple to a list
-  - `:last_working_route` and `:transmission_speed` have been seperated
-  - `:last_working_route` is now a list
-  - `:route_changed` is now a boolean
+  * `:rssi` is now `:rssi_hops` and has been changed from a tuple to a list
+  * `:last_working_route` and `:transmission_speed` have been separated
+  * `:last_working_route` is now a list
+  * `:route_changed` is now a boolean
 
-### Fixes
+### Fixed
 
 * Fix error when sending `Grizzly.ZWave.Commands.StatisticsGet`
 
 ## v0.20.2 - 2021-8-11
 
-### Updates
+### Changed
 
-- Turn off TLS warning for connecting with `zipgateway` server
+* Turn off TLS warning for connecting with `zipgateway` server
 
 ## v0.20.1 - 2021-7-1
 
-### Adds
-
-- Support for ScheduleEntryLock command class (Grace Yanagida)
-  - `Grizzly.ZWave.CommandClasses.ScheduleEntryLock`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockDailyRepeatingGet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockDailyRepeatingReport`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockDailyRepeatingSet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockEnableAllSet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockEnableSet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockTimeOffsetGet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockTimeOffsetReport`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockTimeOffsetSet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockWeekDayGet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockWeekDayReport`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockWeekDaySet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockYearDayGet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockYearDayReport`
-  - `Grizzly.ZWave.Commands.ScheduleEntryLockYearDaySet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryTypeSupportedGet`
-  - `Grizzly.ZWave.Commands.ScheduleEntryTypeSupportedReport`
-
-### Fixes
-
-- Invalid warnings when calling function in `Grizzly.SwitchBinary`
+### Added
 
-### Updates
+* Support for ScheduleEntryLock command class (Grace Yanagida)
```

Comment:
Rather than adding my name here, would it make sense to copy the format of v0.19.1 where it just lists contributors at the end? ie Thank you to those who contributed to this release: * Jean-Francois Cloutier * Grace Yanagida

Reply:
Yeah, I go back and forth on this.

I think I tried this one out so that the name is tied directly to the unit of work that was complete. However, listing the names seems better because I am only been adding work that affects end-users more recently into the changelog. That would some behind-the-scenes work might not directly be mentioned in the release notes, so maybe having the list is better.